sceptre-example/requirements.txt

```diff
@@ -0,0 +1,3 @@
+sceptre
```

Comment:
is this required?  isn't Sceptre already a dependency of sceptre-cdk-handler ?

Reply:
Probably not required, no. But I think folks would want to pin their version of Sceptre in their project anyway and have it as an explicit requirement rather than relying on it being a dependency of a dependency.

Reply:
ok